app/jobs/upvs/fetch_public_authority_edesks_list_job.rb

```diff
@@ -58,6 +54,6 @@ def check_row_attributes(attributes)
   def assert_known_edesks_existence!
     repository = TemporaryPublicAuthorityEdesk
     repository.find_by!(uri: 'ico://sk/00151513', cin: '151513', name: 'Úrad vlády Slovenskej republiky')
-    repository.find_by!(uri: 'ico://sk/00151513_10001', cin: '151513', name: 'Úrad vlády Slovenskej republiky - ÚPVS')
```

Comment:
toto sa preco menilo?

Reply:
Na toto som chcel upozornit. V tom novom datasete sa vobec nenachadza zaznam k uri: 'ico://sk/00151513_10001', cin: '151513', name: 'Úrad vlády Slovenskej republiky - ÚPVS'. Neviem preco.

Reply:
@jsuchal co ty na to?

Reply:
@jsuchal ping

Reply:
Vies spravit diff toho co sa nachadza v novom a co tam chyba oproti staremu?

Reply:
Jasne

Reply:
@jsuchal https://github.com/smith558/Zoznam-OVM-s-aktivovanou-schrankou tu je to v exceli, cervenou su vyznacene zaznamy ktore chybaju pri porovnany s druhym datasetom (navzajom)

Reply:
@smith558 btw existuje https://gist.github.com kde si mozes vytvarat secret gists